config/application.yml.default

```diff
@@ -58,6 +58,7 @@ backup_code_cost: '2000$8$1$'
 broken_personal_key_window_start: '2021-07-29T00:00:00Z'
 broken_personal_key_window_finish: '2021-09-22T00:00:00Z'
 component_previews_enabled: false
+component_previews_embed_frame_ancestors: '["https://developers.login.gov"]'
```

Comment:
I think we should leave this empty by default and override on the deployed boxes.

```suggestion
component_previews_embed_frame_ancestors: '[]'
```